examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb

```diff
@@ -784,7 +793,54 @@
     }
    ],
    "source": [
-    "response = send_triton_request(request, outputs)\n",
+    "# read in data for request\n",
```

Comment:
This code should be injected in to the notebook, not part of the actual notebook. The user shouldn't grab this data from the parquet file. They should get the raw_user_id from part 1 notebook when exploring the users. This is necessary when we are creating synthetic data for unit/integration tests